### ISSUE TITLE
INTRANET - header link underline contrast

### ISSUE THREAD

**Issue by Kamla55** [wet-boew/cdts-sgdc#195]
An accessibility fail for contrast ratio on the links in the header template (IntraWeb main header)

 
![image](https://user-images.githubusercontent.com/71461405/99976756-d7341080-2d71-11eb-903a-b7d7e71c4d11.png)

**Comment by jeanbenoitrochon** [comment]
Adding details:
Accessibility fail for contrast ratio on the links in the header template (IntraWeb)

![195_accessibility](https://user-images.githubusercontent.com/47452736/99997156-c17f1500-2d8a-11eb-8bd6-f819137b5ed7.png)

**Comment by patricehernert** [comment]
@Kamla55 : Hello Kamla, we can not reproduce the problem. Could you give me a full url that is not passing accessibility ?

**Comment by Kamla55** [comment]
Here is the link : http://esdc.prv/en/lb/index.shtml 

And the accessibility reports that identified the issue in Compliance Sheriff
[DPAPSSGDGGITPS_Stat_RapportAccessibilite_20200817_V01.docx](https://github.com/wet-boew/cdts-sgdc/files/5596807/DPAPSSGDGGITPS_Stat_RapportAccessibilite_20200817_V01.docx)

![ComplianceSheriff-templateError](https://user-images.githubusercontent.com/71461405/100231415-f6a77680-2ef4-11eb-9cde-16a80d72eda1.jpg)

**Comment by jeanbenoitrochon** [comment]
Issue was closed by accident, re-opening.

**Comment by patricehernert** [comment]
I changed colour inside "eccc.css" and pushed it to gccode:

<img width="1340" alt="1" src="https://user-images.githubusercontent.com/56038788/101181864-61eef800-361b-11eb-98c4-211dc3a0a3fa.PNG">

**Comment by Kamla55** [comment]
Hang on checking in with the team that reported the issue - they will test and get back to us

**Comment by ahmad-shahid** [comment]
Confirmed to be working.
